gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -754,7 +754,7 @@ end
 -- @realm server
 -- @deprecated
 function plymeta:GetAvoidDetective()
-	return self:GetAvoidRole(ROLE_DETECTIVE)
+	return false
```

Comment:
Remove Role Avoidance code completely. Add it as breaking change.
Dont think anybody would use these.

Reply:
FOr me only the full removal of role avoidance is missing.

Reply:
check again